sdk/python/kfp/dsl/for_loop.py

```diff
@@ -99,25 +125,23 @@ def __init__(
             name_code: A unique code used to identify these loop arguments.
                 Should match the code for the ParallelFor ops_group which created
                 these LoopArguments. This prevents parameter name collisions.
-            name_override: The override name for PipelineParameterChannel.
-            **kwargs: Any other keyword arguments passed down to PipelineParameterChannel.
+            name_override: The override name for PipelineChannel.
+            **kwargs: Any other keyword arguments passed down to PipelineChannel.
         """
         if (name_code is None) == (name_override is None):
             raise ValueError(
                 'Expect one and only one of `name_code` and `name_override` to '
                 'be specified.')
 
         if name_override is None:
-            super().__init__(name=self._make_name(name_code), **kwargs)
+            super().__init__(name=_make_name(name_code), **kwargs)
         else:
             super().__init__(name=name_override, **kwargs)
 
-        if not isinstance(
-                items,
-            (list, tuple, pipeline_channel.PipelineParameterChannel)):
+        if not isinstance(items,
+                          (list, tuple, pipeline_channel.PipelineChannel)):
```

Comment:
Do we still want to enforce that this is a `PipelineParameterChannel` if it's a `LoopParameterArgument`? There may be another reason for loosening this check that I'm not aware of.

Reply:
Done.